bundles/org.pitest.pitclipse.ui/src/org/pitest/pitclipse/ui/view/PitView.java

```diff
@@ -18,28 +18,75 @@
 
 import java.io.File;
 
+import org.eclipse.jface.action.Action;
+import org.eclipse.jface.action.IToolBarManager;
 import org.eclipse.swt.SWT;
 import org.eclipse.swt.browser.Browser;
 import org.eclipse.swt.widgets.Composite;
+import org.eclipse.ui.IActionBars;
 import org.eclipse.ui.part.ViewPart;
 import org.pitest.pitclipse.ui.utils.PitclipseUiUtils;
 
 /**
  * A view displaying the HTML report generated by PIT.
  */
 public class PitView extends ViewPart implements SummaryView {
+    public static final String VIEW_ID = "org.pitest.pitclipse.ui.view.PitView";
+    public static final String BLANK_PAGE = "about:blank";
+    public static final String BACK_BUTTON_TEXT = "<";
+    public static final String HOME_BUTTON_TEXT = "Home";
+    public static final String FORWARD_BUTTON_TEXT = ">";
     private Browser browser = null;
+    private String homeUrlString = null;
 
     @Override
     public synchronized void createPartControl(Composite parent) {
-        PitclipseUiUtils.executeSafely
-            (() -> {
-                browser = new Browser(parent, SWT.NONE);
-                browser.addProgressListener(new PitUiUpdatePublisher(browser));
-            },
-            parent,
-            "Failed to create the Pit Summary view",
-            "Browser cannot be initialized.");
+        PitclipseUiUtils.executeSafely(
+                () -> {
```

Comment:
just cosmetic: we could take an indent back (4 spaces should be enough for indentation, instead of 8)